src/kernel-install/kernel-install.in

```diff
@@ -83,7 +83,7 @@ if [ "${0##*/}" = "installkernel" ]; then
     # kernel's install.sh invokes us as
     #   /sbin/installkernel <version> <vmlinuz> <map> <installation-dir>
     # We ignore the last two arguments.
-    set -- "$1"
+    set -- "$1" "$2"
```

Comment:
hmm, to be more accurate, shouldn't this move further up, before we look for --help/-h/--version/--verbose/-v?

I mean those switches are not defined for "/sbin/installkernel" afaik, and one could argue that if passed anyway they should be understood as version parameter?

btw, should we refuse if less than 2 args are specified? simply to catch cases if people invoke the script under that name directly to see what it does?

Reply:
> hmm, to be more accurate, shouldn't this move further up, before we look for --help/-h/--version/--verbose/-v?
>
> I mean those switches are not defined for "/sbin/installkernel" afaik, and one could argue that if passed anyway they should be understood as version parameter?

I think it's nicer to allow `--version`, `--verbose`, and `--help` to work with `installkernel`. The original script didn't support that, but we can. If somebody names their kernel `--help`, that's not our problem.
 
> btw, should we refuse if less than 2 args are specified? simply to catch cases if people invoke the script under that name directly to see what it does?

We should. I added a check now.